fdbserver/storageserver.actor.cpp

```diff
@@ -8558,6 +8558,10 @@ ACTOR Future<Void> update(StorageServer* data, bool* pReceivedUpdate) {
 				} else {
 					MutationRef msg;
 					cloneReader >> msg;
+					if (isEncryptionOpSupported(EncryptOperationType::TLOG_ENCRYPTION) && !msg.isEncrypted() &&
+					    !msg.param1.startsWith(backupLogKeys.begin) && !msg.param1.startsWith(applyLogKeys.begin)) {
```

Comment:
if `msg` is a clear range, do we also want to check `param2`?

Reply:
Normal clear range mutations should be encrypted with our current logic so i don't think we need to make an exception for it. `blog` and `alog` SetValues are written to the Tlog in a separate place than regular mutations so that is why we need to exclude it

Reply:
something like this?
```suggestion
					    !(isSingleKeyMutation(msg) && (backupLogKeys.contains(msg.param1) || (applyLogKeys.contains(msg.param1))) {
```